src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -87,6 +87,12 @@ public abstract class RemoteAuthenticationHandler<TOptions> : AuthenticationHand
             return true;
         }
 
+        /// <summary>
+        /// Authenticate the user identity with the identity provider. 
+        ///
+        /// This could be done through a back channel communication with the identity provider. Exception thrown during
+        /// the authenticating should be saved to the AuthenticateResult.
```

Comment:
I still think the try/catch should be moved here, the whole point of this class being created was to streamline the error/exception handling of Auth Handlers.  With this comment, we are basically saying its pretty much a bug if they don't have a try/catch/Fail in their implementation of `HandleRemoteAuthenticateAsync`... which at that point we might as well do the Try/Catch/Fail when we call this method so its foolproof.  That is my main argument.